File: problemas/iniciante/3040/3040.ml
Scanf.scanf "%d\n" (fun n ->
    for _ = 1 to n do
      Scanf.scanf "%d %d %d\n" (fun altura espessura galhos ->
          if 200 <= altura && altura <= 300 && 50 <= espessura && 150 <= galhos
          then Printf.printf "Sim\n"
          else Printf.printf "Nao\n")
    done)

File: problemas/iniciante/3040/3040.go
package main

import (
    "fmt"
)

func main() {
    var n, altura, espessura, galhos int

    fmt.Scanf("%d", &n)

    for i := 0; i < n; i++ {
        fmt.Scanf("%d %d %d", &altura, &espessura, &galhos)
        if 200 <= altura && altura <= 300 && 50 <= espessura && 150 <= galhos {
            fmt.Printf("Sim\n")
        } else {
            fmt.Printf("Nao\n")
        }
    }
}
